modules/compiler/utils/source/lld_linker.cpp

```diff
@@ -59,11 +59,11 @@ Expected<std::unique_ptr<MemoryBuffer>> lldLinkToBinary(
   SmallString<128> elfFileName;
   SmallString<128> linkerScriptFileName;
   SmallString<128> linkRTFileName;
-  sys::fs::createTemporaryFile("lld", "o", objFileName);
-  sys::fs::createTemporaryFile("lld", "elf", elfFileName);
-  sys::fs::createTemporaryFile("lld", "ld", linkerScriptFileName);
+  (void)sys::fs::createTemporaryFile("lld", "o", objFileName);
```

Comment:
I wonder whether we should just check for failure here - but it's no worse than before, so up to you.

Reply:
Oh right, we can actually indicate failure here, will update.

Reply:
looks good